src/components/Class/ClassList/index.js

```diff
@@ -14,57 +14,55 @@ function ClassList() {
   useEffect(() => {
     dispatch(classActions.getClasses());
   }, [dispatch]);
+
   if (loading) {
     return <Loader pageLoader={true} />;
   }
+  const obj = {
+    hi: "Hindi",
+    te: "Telugu",
+    en: "English",
+    ta: "Tamil",
+    doubt_class: "Doubt Class",
+    workshop: "Workshop",
+  };
 
   return (
     <>
-      <table>
-        <thead>
-          <tr>
-            <th>Facilitator Name</th>
-            <th>Title</th>
-            <th>Description</th>
-            <th>Language</th>
-            <th>Class type</th>
-            <th>Date</th>
-            <th> Start Time </th>
-            <th>End Time</th>
-          </tr>
-        </thead>
-
+      <div className="ng-upcoming-class">
         {data && data.length > 0 ? (
           data.map((item, index) => {
             const classStartTime =
               item.start_time && item.start_time.replace("Z", "");
             const classEndTime =
               item.end_time && item.end_time.replace("Z", "");
             return (
-              <tr key={index}>
-                <td data-column="Facilitator Name"> {item.facilitator.name}</td>
-                <td data-column="Title">{item.title}</td>
-                <td data-column="Description">{item.description}</td>
-                <td data-column="Language">{item.lang}</td>
-                <td data-column="Class type">{item.type}</td>
-                <td data-column="Date">
-                  {moment(classStartTime).format("DD-MM-YYYY")}
-                </td>
-                <td data-column="End Time">
-                  {moment(classStartTime).format("hh:mm a")}
-                </td>
-                <td data-column="End Time">
-                  {moment(classEndTime).format("hh:mm a")}
-                </td>
-              </tr>
+              <div key={index} className="class-cards">
```

Comment:
Could we move to a new component? something like `ClassCard` and pass the class data into that. 
It might look like this. `<ClassCard item={item} />` Let me know your views?

Reply:
Done